File: projects/m1/004-units-of-time/solutions/GianlucaGuerra/solution_units_of_time.py
#counter from start a session on PC
#number of days-hours-minutes-seconds
print ("Please insert the amount of: ")
a=int(input("Days: "))
b=int(input("Hours: "))
c=int(input("Minutes: "))
d=int(input("Seconds: "))
total= (a*86400)+(b*3600)+(c*60)+(d) 
print("Total of your time in seconds is: " + str(total))

File: projects/m1/004-units-of-time/solutions/WalterdeBenedittis/solution-units-of-time.js
    let insDays = prompt("Insert the days of your date")*86400;
    let insHours = prompt("Insert the hours of your date")*3600;
    let insMinutes = prompt("Insert the minutes of your date")*60;
    let insSeconds = prompt("Insert the seconds of your date");

    let totalSec = insDays + insHours + insMinutes + insSeconds
    alert("This is your date in Seconds: " + totalSec); 
